File: PYTHON PROGRAMMING/Test7.py
# 7. SWAPPING OF TWO VARIABLES' VALUES USING A THIRD VARIABLE
input1 = int(input("Enter Input1: "))
input2 = int(input("Enter Input2: "))
temp = 0

print(f"Current Input1 Value: {input1}")
print(f"Current Input2 Value: {input2}")

temp = input2
input2 = input1
input1 = temp

print(f"Updated Input1 Value: {input1}")
print(f"Updated Input2 Value: {input2}")

File: JAVASCRIPT PROGRAMMING/Test7.js
"use strict";
// 7. SWAPPING OF TWO VARIABLES' VALUES USING A THIRD VARIABLE
const tempSwap = (ip1, ip2) => {
    console.log(`Current Num1 Value: ${ip1}`);
    console.log(`Current Num2 Value: ${ip2}`);
    let temp = 0;
    temp = ip1;
    ip1 = ip2;
    ip2 = temp;
    console.log(`Updated Num1 Value: ${ip1}`);
    console.log(`Updated Num2 Value: ${ip2}`);
};
tempSwap(10, 20);